modules/tools/mobileye_viewer/location_server.py

```diff
@@ -19,7 +19,7 @@
 import os
 import rospy
 import math
-import thread
+import _thread
```

Comment:
The next step is to fix the header orders to satisfy coding style requirement. Maybe with tools like autopep8.